org.gnome.Connections.json

```diff
@@ -80,7 +84,7 @@
                 {
                     "type" : "git",
                     "url" : "https://gitlab.gnome.org/gnome/gtk-frdp.git",
-                    "branch" : "gtk-frdp-0-1"
+                    "branch" : "a44d84ec687f41ad37e43697b28a018f65780780"
```

Comment:
you sould be using `commit` then

Reply:
Are they not equivalent?

Reply:
from the point of view of the manifest yes, from the PoV of readability / semantics ,no.

I personally which the presence of "branch" was a failure on flathub, in the name or reproducibility